### PR TITLE
import into: rename pkg dir

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #42930

Problem Summary:

### What is changed and how it works?
- rename package to avoid mixing with load data
- add an empty `importintotest` real-tikv-test, will enable and move `loaddatatest` into it later

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code
  - there's no code logic change in this pr

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -517,6 +517,10 @@ bazel_loaddatatest: failpoint-enable bazel_ci_prepare
 	bazel $(BAZEL_GLOBAL_CONFIG) test $(BAZEL_CMD_CONFIG) --test_arg=-with-real-tikv --define gotags=deadlock,intest \
 		-- //tests/realtikvtest/loaddatatest/...
 
+bazel_importintotest: failpoint-enable bazel_ci_prepare
```
Comment (lance6716) [inline Makefile:520]: @hawkingrei what change should we make to let CI run it?

Comment (D3Hunter) [reply Makefile:520]: see https://github.com/PingCAP-QE/ci/pull/2157

Comment (hawkingrei) [reply Makefile:520]: first, You should create a new empty with-real-tikv test. then you need to enable it in the CI.  finally, we can merge this PR.

Comment (D3Hunter) [reply Makefile:520]: after discuss @hawkingrei , will create an empty with-real-tikv test `importintotest` in this pr, after enable it in the CI, will move code in `loaddatatest` into `importintotest`

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**